src/leaflike/routes.clj

```diff
@@ -0,0 +1,19 @@
+(ns leaflike.routes
+  (:require [bidi.ring :as bidi]
+            [ring.util.response :as res]))
+
+(defn welcome [_]
+  (res/response "Welcome to leaflike"))
+
+(defn yay [_]
+  (res/response "Yay!"))
+
+(defn ping [request]
+  (res/response {:ping (-> request :route-params :ping)}))
```

Comment:
Just FYI this will **not** return a valid JSON.

Reply:
yes, we need to add the middleware to return the valid json.